lang_python/parsing/Parser_python.mly

```diff
@@ -272,8 +270,11 @@ import_from:
       { [ImportFrom ($1, $2, $4)] }
 
 name_and_level:
-  |           dotted_name { $1, None }
```

Comment:
oops, good catch.

Reply:
Similarly here, I made sure that the generated AST was identical before and after this change.